operator/pkg/reconciliation/check_nodes.go

```diff
@@ -25,208 +27,189 @@ func (rc *ReconciliationContext) GetPVCForPod(podNamespace string, podName strin
 	return pvc, nil
 }
 
-func (rc *ReconciliationContext) DeletePvcIgnoreFinalizers(podNamespace string, podName string) (*corev1.PersistentVolumeClaim, error) {
-	var wg sync.WaitGroup
+func (rc *ReconciliationContext) getPodsPVCs(pods []*corev1.Pod) ([]*corev1.PersistentVolumeClaim, error) {
+	pvcs := []*corev1.PersistentVolumeClaim{}
+	for _, pod := range pods {
+		pvc, err := rc.GetPodPVC(pod.Namespace, pod.Name)
+		if err != nil {
+			return nil, err
+		}
+		pvcs = append(pvcs, pvc)
+	}
+	return pvcs, nil
+}
 
-	wg.Add(1)
+func (rc *ReconciliationContext) getNodesForNameSet(nodeNameSet map[string]bool) ([]*corev1.Node, error) {
```

Comment:
Let's not do this now, but in the future, we should probably define a custom type for NameString of map[string]bool

Reply:
map[string]bool is StringSet though?

Reply:
Oi! Sorry, missed the ones in this file. Will fix.